api.js

```diff
@@ -195,7 +195,7 @@ function handlePaths(files) {
 		files = [
 			'test.js',
 			'test-*.js',
-			'test/*.js'
```

Comment:
I don't think we should make this particular change. It changes the default behavior of simply running `ava`. The default behavior should remain the same. If they want to use this feature they should have to run `ava test`.